### ISSUE TITLE
Input Parameters: 'Add Param' and close(X) icon' controls present under 'Input Parameters' blade are not accessible through keyboard.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1047]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242476

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure.
Navigate to "New Stored Procedure" menu item and select it.
Navigate to Stored Procedure ID edit field and give input and select Save menu item.
Now Navigate to "Execute" control and select it.
Navigate through all the controls under "Input Parameters" blade.
Navigate to Param edit edit field and again press tab key to navigate to 'Add Param' and close(X) icon' controls.
Verify the issue.

Actual Result:
'Add Param(+)' and Close(X) icon' controls present under 'Input Parameters' blade are not accessible through keyboard.

Expected Result:
'Add Param' and Close(X) icon' controls present under 'Input Parameters' blade which are accessible using mouse should also accessible through keyboard.

https://user-images.githubusercontent.com/8519407/132716884-793049b1-3cbb-4686-b3a4-0c1003dd9134.mp4
